docs/modules/wan/pages/tuning.adoc

```diff
@@ -754,3 +754,47 @@ hazelcast:
         use-endpoint-private-address: true
 ----
 ====
+
+== Replicating `IMap`/`ICache` Evictions
+
+The replication of `IMap` and `ICache` evictions to other clusters can be enabled by setting the relevant xref:ROOT:system-properties.adoc[Hazelcast
+system property] to `true`. The replication of eviction events can be useful for maintaining stronger data consistency
+between clusters, however it should be used with caution: source clusters with eviction enabled should replicate to
+target clusters which **do not** have eviction enabled. If both source and target clusters have eviction enabled, due
+to eviction being a largely local operation, it is possible for a target cluster to evict data that has not been
+evicted on the source cluster. When replication is used for data consistency on a failover deployment, it is important
+that eviction is re-enabled on the standby cluster when it becomes active.
+
+While enabling this functionality can help to maintain stronger data consistency over WAN between clusters, it is
+important to note that this **does not guarantee full data consistency**.
```

Comment:
Awesome content @JamesHazelcast , thanks a lot.
I'd suggest to move the very first sentence of this section to here (right before the configuration examples), with a minor editing:

"You can enable the replication of `IMap` and `ICache` evictions to the other clusters by setting the `hazelcast.wan.replicate.imap/icache.evictions` property to `true`, as shown below."